lib/src/Base/Common/openturns/PersistentObject.hxx

```diff
@@ -151,6 +151,13 @@ public:
     return OSS() << offset << __repr__();
   }
 
+  /* String converter */
+  inline
```

Comment:
```suggestion
```

Reply:
I do not understand this suggestion. The `__repr__()`is `inline`:
https://github.com/openturns/openturns/blob/31fc1f66f383304a354eafd6bb674ce0dc1c9260/lib/src/Base/Common/openturns/PersistentObject.hxx#L141
So why should'nt be `__repr_html__` be inline too?

Reply:
Fixed.